libs/tx-builder/src/utils/args.ts

```diff
@@ -86,6 +86,21 @@ export const processArg = async ({
   if (arg?.type === 'static') {
     return arg.value;
   }
+  if (arg?.type === 'template') {
+    // appState variables should be enclosed in curly braces e.g. `Send {.formValues.value} ETH`
```

Comment:
Not quite following this comment.

Could you drop an example of that this looks like in a txLego?

Reply:
here's an example argument that can be added to a txLego using the template argType
```
{
          type: 'template',
          value:
            'Signal Proposal sent with offering {.formValues.proposalOffering} wei',
},
```